Sources/SwiftInspectorAnalyzers/Tests/StaticUsageAnalyzerSpec.swift

```diff
@@ -184,7 +184,6 @@ final class StaticUsageAnalyzerSpec: QuickSpec {
         let content = """
         JitneyProducer.shared.publish(
               PaidGrowth.V1.PaidGrowthSignupCompletePixelEvent(
-                // TODO: Use the globalLoggingContext property on an injected LoggingService than accessing a singleton directly.
```

Comment:
This change isn't related to this PR but I was searching for TODOs and this one was distracting 😅